### ISSUE TITLE
"prompt weighting" should be renamed to "prompt interpolation" for v2.0 release

### ISSUE THREAD

**Issue by damian0815** [invoke-ai/InvokeAI#931]
Since it's unlikely that attention weighting (ref comments on #917) will make it into the v2 release, the existing "prompt weighting" feature should be renamed in documentation to "prompt interpolation" or "prompt interpolation weighting" to avoid users confusing it with the more-likely expected prompt *attention* weighting.

**Comment by Any-Winter-4079** [comment]
I like the term `prompt interpolation` for the feature. The other option seems a bit too wordy for me (and thinking of an end user too)

**Comment by Any-Winter-4079** [comment]
In fact, we could call the two features `prompt attention` and `prompt interpolation`, and leave the term `weighting` out of the names, since in reality we may use weighting for both techniques, but it's just a personal opinion.

**Comment by damian0815** [comment]
what about 
`prompt tweening`
and
`attention weighting`?

**Comment by Any-Winter-4079** [comment]
Sounds good to me!

**Comment by damian0815** [comment]
@bmaltais what do you think?

**Comment by damian0815** [comment]
fwiw i also think the terminology "sub-prompt" adds additional confusion - the prefix "sub-" sets up a hierarchical expectation in the language that i feel isn't there in the math.

**Comment by damian0815** [comment]
another candidate is `prompt mixing` which makes it perhaps more approachable for non-technical users. everybody knows if you mix red and blue you make green, so they shouldn't be surprised that if you mix "fish" and "blanket" you get neither one nor the other but something weird and perhaps unexpected.

**Comment by lstein** [comment]
To my ear, one of `prompt mixing`, `prompt blending` or `prompt merging` gives the right sense of what's going on.

**Comment by lstein** [comment]
The next tagged release candidate will refer to `prompt blending`, and I'll push it as soon as I complete a few other bug fixes. I'll post a link to the revised docs for review.

**Comment by lstein** [comment]
Here's a link to the revised documentation. Let me know if you think it's a decent guide to what to expect: 

[Prompt Blending](https://github.com/invoke-ai/InvokeAI/blob/release-candidate-2/docs/features/PROMPTS.md#prompt-blending)

**Comment by damian0815** [comment]
@lstein this is 🔥 . i really like "exploring the mind of the AI". and i see what you did with sneakily adding the "hybrid" at the end - only tweak i would do is to add examples showing "blue sphere:0.4 red cube:0.6" and "blue sphere:0.6 red cube:0.4" both without the "hybrid" word, since these cross-concept blends often go wild around the `0.5` interpolation point.

**Comment by lstein** [comment]
> these raw cross-concept blends often go wild around the 0.5 interpolation point but are generally more stable on either side of it.

I didn't know that! Looks like I'm going to sink a few more hours into exploring prompt blending now that I understand (sort of) what's going on.

**Comment by bmaltais** [comment]
100% agree.
